agent/api/task.go

```diff
@@ -242,34 +242,34 @@ func (task *Task) UpdateMountPoints(cont *Container, vols map[string]string) {
 	}
 }
 
-// updateTaskKnownState updates the given task's status based on its container's status.
+// updateTaskKnownStatus updates the given task's status based on its container's status.
 // It updates to the minimum of all containers no matter what
-// It returns a TaskStatus indicating what change occured or TaskStatusNone if
+// It returns a TaskStatus indicating what change occurred or TaskStatusNone if
 // there was no change
+// Invariant: task known status is the minimum of container known status
 func (task *Task) updateTaskKnownStatus() (newStatus TaskStatus) {
-	llog := log.New("task", task)
-	llog.Debug("Updating task")
+	seelog.Debug("Updating task %v", task)
```

Comment:
I'm not a fan of dumping a whole struct into the logs. At least this is at debug severity, but if we can limit the output to fields that are actually going to be meaningful, it would be preferable.